.azure/pipelines/quarantined-pr.yml

```diff
@@ -10,18 +10,6 @@ trigger:
     include:
     - master
     - release/5.0
-  paths:
-    exclude:
-    - .github/*
-    - .vscode/*
-    # Exclude only a few specific Markdown files because some of them ship.
-    - docs/*
-    - CODE-OF-CONDUCT.md
-    - CONTRIBUTING.md
-    - LICENSE.TXT
-    - README.md
-    - SECURITY.md
-    # Do not ignore the THIRD-PARTY-NOTICES.TXT file because it ships.
```

Comment:
I'm on the fence here. Removed the exclusions because extra builds mean the quarantined tests are run more often. But, this uses some resources&hellip;

Any other thoughts @dotnet/aspnet-build❔

Reply:
I'd prefer to keep the exclusions here, since quarantined-test isn't a required check. No use in wasting the resources.

Reply:
This isn't in the PR rule. The exclusions _do_ exist [there](https://github.com/dotnet/aspnetcore/pull/29321/files#diff-97201ea9bd0e7bec67d1e4f33b422e44aacc3f59d5c591106d5b51222f9a9953R21-R31). This is just the triggered builds, which may occur after a doc-only change is merged but is often combined with other changes.

Reply:
Put another way, I left the PR exclusions alone **because** this pipeline isn't a required check and that reduces waste. I removed the trigger exclusions because the number of relevant builds should be smaller but the coverage helps.